tests/output/stats.log

```diff
@@ -0,0 +1,90 @@
+test:skydive_tests/web-stress-6, time:27/11/2018 16:05:1543334713, TEST_STARTING, min_runtime:300s, iperf_bw:, min_flows_per_generator:100000, delay:2ms
```

Comment:
Please remove all of those files from the PR as those are specific to a run and we do not want to include them in git